Packages/KeyAppKit/Sources/BankTransfer/Service/StrigaMockBankTransferService.swift

```diff
@@ -12,12 +14,25 @@ public final class StrigaMockBankTransferService: BankTransferService {
     public init() { }
 
     public func reload() async {
-        let data = UserData(countryCode: "fr", userId: nil, mobileVerified: false)
+        sleep(5)
```

Comment:
Don't use sleep as it will terminate the app. Let's use `await Task.sleep(nanoseconds: 1_000_000_000)` to terminate only the current task and faking network request 1 sec (for example)